doc/python/getting-started.md

```diff
@@ -66,17 +66,15 @@ or `conda`:
 ```
 $ conda install -c plotly plotly=5.24.1
 ```
-This package contains everything you need to write figures to standalone HTML files.
 
-> Note: **No internet connection, account, or payment is required to use plotly.py.** Prior to version 4, this library could operate in either an "online" or "offline" mode. The documentation tended to emphasize the online mode, where graphs get published to the Chart Studio web service. In version 4, all "online" functionality was removed from the `plotly` package and is now available as the separate, optional, `chart-studio` package (See below). **plotly.py version 4 is "offline" only, and does not include any functionality for uploading figures or data to cloud services.**
+If you want to use Plotly Express, install its required dependencies with:
 
-
-```python
-import plotly.express as px
-fig = px.bar(x=["a", "b", "c"], y=[1, 3, 2])
-fig.write_html('first_figure.html', auto_open=True)
+```
+pip install plotly[express]==5.24.1
```

Comment:
It's a good question. I suppose it guarantees a user gets the version the docs are about when they install, but it would be nice to have it auto-update.

Reply:
In any case, if we keep the version pin, it should be `6.0.0` for this docs update right?

Reply:
Actually I may argue that we should keep the pin for a while.
Namely if we want to use the RC version, without the pip the latest v5 would be installed not the `6.0.0-rc.0` version.

Reply:
True — it depends on whether we are releasing these docs at the time the RC is released, or waiting until the full `6.0.0` release.

Reply:
We won't release the docs with the RC as we don't have proper versioning to manage it.